public/main/main-window.js

```diff
@@ -80,6 +83,7 @@ function loadWindow () {
     backgroundColor: '#323232',
     webPreferences: {
       nodeIntegration: false,
+      contextIsolation: false,
```

Comment:
Default is `false`. Is this needed?

Reply:
[Electron team is planning on changing the defaults for some options in next version (5)](https://electronjs.org/blog/electron-4-0#deprecations) so a warning is thrown in current version if the values of those options are not explicitly defined.

Reply:
So we should set it to `false` explicitly.